runners/flink/job-server/flink_job_server.gradle

```diff
@@ -90,6 +90,7 @@ dependencies {
   runtime project(":sdks:java:io:kafka")
   runtime library.java.kafka_clients
   runtime project(":sdks:java:io:google-cloud-platform")
+  runtime project(path: flinkRunnerProject, configuration: "miniCluster")
```

Comment:
Won't that include `flink-runtime-web` in the release shadow jar? We would want to avoid that because it is provided by the Flink cluster.

Reply:
Yes. I think I have to make a separate jar anyway because including `PortablePipelineJarUtils` in the mini cluster jar seems to break the loading of the pipeline. I haven't yet figured out how to create multiple shadow jars with different configurations, however.

Reply:
Alright, I've updated this PR to build a separate miniCluster jar that includes/excludes the correct stuff. PTAL